test/unit/org/apache/cassandra/db/guardrails/GuardrailsTest.java

```diff
@@ -40,13 +40,13 @@ public class GuardrailsTest extends GuardrailTester
     public static final int DISABLED = -1;
 
     @Test
-    public void testDisabledThreshold() throws Throwable
+    public void testDisabledMaxThreshold() throws Throwable
     {
         Threshold.ErrorMessageProvider errorMessageProvider = (isWarn, what, v, t) -> "Should never trigger";
-        testDisabledThreshold(new Threshold("x", state -> DISABLED, state -> DISABLED, errorMessageProvider));
+        testDisabledMaxThreshold(new MaxThreshold("x", state -> DISABLED, state -> DISABLED, errorMessageProvider));
     }
 
-    private void testDisabledThreshold(Threshold guard) throws Throwable
+    private void testDisabledMaxThreshold(MaxThreshold guard) throws Throwable
```

Comment:
We can leave `testDisabledThreshold(Threshold)` as it was and just call it from `testDisabledMaxThreshold()` and `testDisabledMinThreshold` with a `MaxThreshold` and a `MinThreshold` instance.